kso_utils/tutorials_utils.py

```diff
@@ -433,7 +433,7 @@ def extract_clips(
         full_prompt = init_prompt
         mod_prompt = ""
         output_prompt = ""
-        def_output_prompt = f".output('{str(output_clip_path)}', ss={str(upl_second_i)}, t={str(clip_length)}, movflags='+faststart', crf=20, pix_fmt='yuv420p')"
+        def_output_prompt = f".output('{str(output_clip_path)}', ss={str(upl_second_i)}, t={str(clip_length)}, movflags='+faststart', pix_fmt='yuv420p')"
```

Comment:
The crf is the amount of compression we do. (in my branch extract_modify_clips I had set it to 17 to have no visual change). But why did you remove it here?

Reply:
I see that you added it later again, maybe squash the arguments so that it is not visible in the history.

Reply:
If you go to the "Files changed" tab on the right you will see only the overall changes as well.

Reply:
Yes, I know. But when PRs are large it is hard to follow all the changes there. It is easier to follow if every commit has a short explanation with the reasoning. Then one can also see why it is changed and this helps a lot for looking back in the history later as well.